client/src/components/Sidebar/index.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import 'antd/dist/antd.css';
+import './style.css';
+import { Layout, Menu } from 'antd';
+import {
+  DesktopOutlined,
+  PieChartOutlined,
+  FileOutlined,
+  LogoutOutlined,
+  MenuFoldOutlined,
+} from '@ant-design/icons';
+import { arrayOf, element, func } from 'prop-types';
+import logoImage from '../../assets/images/logo.png';
+
+const { Header, Content, Footer, Sider } = Layout;
+function Sidebar({ contentComponents, logoutFunction }) {
+  const [collapsed, setCollapsed] = useState(false);
+  const [itemKey, setItemKey] = useState('1');
+
+  const onCollapse = (collapsedValue) => setCollapsed(collapsedValue);
+
+  return (
+    <Layout className="page-layout">
+      <Sider collapsible collapsed={collapsed} onCollapse={onCollapse}>
+        <div className={collapsed ? 'logo logo-hidden' : 'logo'}>
+          <div>
+            <img src={logoImage} alt="" />
+          </div>
+          <h3 className="title-text">Dental Clinic</h3>
```

Comment:
Use the `Title` component from the `antd`

Reply:
Done